rotkehlchen/chain/ethereum/modules/liquity/trove.py

```diff
@@ -226,13 +226,20 @@ def _query_deposits_and_rewards(
                     usd_value=asset_price * amount,
                 ),
             )
+            # `proxy_to` is not accessed from any other location.
+            data[current_address]['proxy_to'] = self.ethereum.proxies_inquirer.proxy_to_address.get(current_address)  # type: ignore  # noqa: E501
```

Comment:
But this is a really bad solution. You are ignoring the type and that should let you know that something is wrong. This function returns a mapping of address to assetbalance and you are breaking that constraint with this change. Basically you made useless all the typing for functions that call this one.

What you can do and I find better is in the two functions that use this one (`get_stability_pool_balances` and `liquity_staking_balances`) change the type to 

```python
dict[ChecksumEvmAddress, dict[str, Any]]
```

and in there do:

```python
for address in addresses:
    staking_data['proxy_to`] = self.ethereum.proxies_inquirer.proxy_to_address.get(current_address)
```

Why I find this better than your first approach:

1. You are not initializing new objects that you don't use
2. You are not reading and settings properties that you don't need

Why I see it better than this approach:

1. You don't break the types everywhere

Reply:
> dict[ChecksumEvmAddress, dict[str, Any]]

This is no different from adding a `type: ignore` and as a result the assertions made above as regards rendering typing redundant also holds.

The only +ve to this is that wherever the function is called, you won't expect the key-value pairs to have the same types.

Reply:
Yes, for the linter is no different but for the reader now it won't expect it to be a mapping for balances